### PR TITLE
minor: fix headers in various files not controlled by CI

### PR DESCRIPTION
**Pull request by rnveach**

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 // checkstyle: Checks Java source code for adherence to a set of rules.
-// Copyright (C) 2001-2014  Oliver Burn
+// Copyright (C) 2001-2020 the original author or authors.
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java]: Can we remove content of this line at all, we do not need to update this file each year.

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputEmptyLineSeparator.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 // checkstyle: Checks Java source code for adherence to a set of rules.
-// Copyright (C) 2001-2014  Oliver Burn
+// Copyright (C) 2001-2020 the original author or authors.
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputEmptyLineSeparator.java]: I think we can make this line empty too

File: src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 // checkstyle: Checks Java source code for adherence to a set of rules.
-// Copyright (C) 2001-2014  Oliver Burn
+// Copyright (C) 2001-2020 the original author or authors.
```
Comment (romani) [inline src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g]: Can you update command at https://github.com/checkstyle/checkstyle/wiki/How-to-make-a-release#before-starting to find and "*.g" files also

Comment (rnveach) [reply src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g]: To confirm, is this correct:
    ```
    find . -type f \( -name *.java -o -name *.header -o -name *.g \) -exec sed -i  \
      "s/\/\/ Copyright (C) 2001-2015 the original author or authors./\/\/ Copyright (C) 2001-2016 the original author or authors./g" \
      {} +
    ```

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java (hunk 1 above)
Comment (rnveach) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java]: @romani do you want me to remove header from all Input files that don't need it? There are more if you look at diff from my update.

https://github.com/checkstyle/checkstyle/commit/ae9edbd4a8645c8cc1a99ad6890dd0006700d1cb#diff-c04011f5de5dd3db31ac528406e0b465R3

**Comment by romani** [comment]
there are only this cases for big header:
```
 ~/java/github/checkstyle/checkstyle/src/it/resources [master|✔] 
11:23 $ ag -Q "Copyright (C)"
com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputEmptyLineSeparator.java
3:// Copyright (C) 2001-2014  Oliver Burn

com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java
3:// Copyright (C) 2001-2014  Oliver Burn

```

File: src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g (hunk 3 above)
Comment (romani) [reply src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g]: yes, correct.
I updated wiki.

Comment (romani) [reply src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g]: I think it make sense to move it `.ci` folder as script to ease operation by multiple maintainers.
current and previous year is easy to get by shell commands.

**Comment by rnveach** [comment]
>  ~/java/github/checkstyle/checkstyle/src/it/resources 

You are only looking in IT. I did a search on all checkstyle for `Copyright (C)` and found 32 matches.

**Comment by romani** [comment]
I do not think that Inputs should have "Copyright", we can remove/cut them (headers), to not confuse us in future.

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputEmptyLineSeparator.java (hunk 2 above)
Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/InputEmptyLineSeparator.java]: removed.

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java (hunk 1 above)
Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputEmptyLineSeparator.java]: removed.

**Pull request merged**